packages/smart-wallet/src/invitations.js

```diff
@@ -52,12 +54,16 @@ export const makeInvitationsHelper = (
   const invitationGetters = /** @type {const} */ ({
     /** @type {(spec: ContractInvitationSpec) => Promise<Invitation>} */
     contract(spec) {
+      fit(harden(spec), shape.ContractInvitationSpec);
+
       const { instance, publicInvitationMaker, invitationArgs = [] } = spec;
       const pf = E(zoe).getPublicFacet(instance);
       return E(pf)[publicInvitationMaker](...invitationArgs);
     },
     /** @type {(spec: PurseInvitationSpec) => Promise<Invitation>} */
     async purse(spec) {
+      fit(harden(spec), shape.PurseInvitationSpec);
```

Comment:
Why is the harden here instead of required of the argument?  It woudl not have been passable without that, and the guard will check that.

Reply:
sgtm, will remove